devops-steps/src/main/java/io/fabric8/kubernetes/pipeline/devops/elasticsearch/ElasticsearchClient.java

```diff
@@ -134,20 +134,22 @@ public static boolean updateEvent(String id, String json, String index, TaskList
         }
     }
 
-    public static boolean isUrlReachable(String url){
-        int timeout = Integer.parseInt(Systems.getEnvVarOrSystemProperty("ES_TIMEOUT","1000")); // default to 1 second
+   public static boolean isUrlReachable(String url) {
+        int timeout = Integer.parseInt(Systems.getEnvVarOrSystemProperty("ES_TIMEOUT", "1000")); // default to 1 second
+        final boolean followRedirectsOriginalValue = HttpURLConnection.getFollowRedirects();
         try {
             HttpURLConnection.setFollowRedirects(false);
```

Comment:
So a better approach would be to use [`setInstanceFollowRedirects`](http://docs.oracle.com/javase/7/docs/api/java/net/HttpURLConnection.html#setInstanceFollowRedirects(boolean)) on the individual connection, rather than this. This approach is still vulnerable to other `HttpUrlConnection`s going at the same time.